final-project/modules/python/onboarding/router.py

```
@@ -0,0 +1,167 @@
+        """Check if the application has been initialized."""
+        is_initialized = await settings_store.get_setting(db, "is_initialized")
+        return OnboardingResponse(
+            success=True,
+            is_initialized=is_initialized == "true",
+            message="initialized" if is_initialized == "true" else "not_initialized"
+        )
+
+    @router.post("/initialize")
+    async def initialize(
+        request: InitializeRequest,
+        db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)